style.css

```diff
@@ -1,10 +1,29 @@
+html,
+body {
+  width: 100vw;
+  height: 100vh;
+  overflow: hidden;
+  background-color: black;
+  margin: 0;
+}
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+div#container {
+  width: 100vw;
+  height: calc(480/640 * 100vw);
+  max-height: 100vh;
+  max-width: calc(640/480 * 100vh);
```

Comment:
The "height" and "max-width" elements don't technically need to be calculated; this is just for readability and ease of changing the aspect ratio.